kibicara/platforms/email/webapi.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (C) 2020 by Maike <maike@systemli.org>
+#
+# SPDX-License-Identifier: 0BSD
+
+from fastapi import APIRouter, Depends, HTTPException, status
+from kibicara.platforms.email.bot import spawner
+from kibicara.platforms.email.model import Email, EmailSubscribers
+from kibicara.platformapi import Message
+from kibicara.config import config
+from kibicara.email import send_email
+from kibicara.model import Hood
+from kibicara.webapi.hoods import get_hood
+from ormantic.exceptions import NoMatch
+from pydantic import BaseModel
+from sqlite3 import IntegrityError
+from nacl.encoding import URLSafeBase64Encoder
+from nacl.secret import SecretBox
+from os import urandom
+
+
+class BodyMessage(BaseModel):
+    """ This model shows which values are supplied by the MDA listener script. """
+    text: str
+    to: str
+    author: str
+    secret: str
+
+
+class Subscriber(BaseModel):
+    """ This model holds the email address of a fresh subscriber. """
+    email: str
+
```

Comment:
Not sure if this is necessary. The subscriber email could probably passed to email_subscribe in an easier way.